bodhi/server/validators.py

```diff
@@ -1036,7 +1036,7 @@ def validate_request(request):
         return
     if request.validated['request'] is UpdateRequest.stable:
         target = UpdateStatus.stable
-    elif request.validated['request'] is UpdateRequest.testing:
+    elif request.validated['request'] is (UpdateRequest.testing or UpdateRequest.batched):
```

Comment:
I noticed that there are some places where you added or statements between ```testing``` and ```batched```, where I would expect ```batched``` to make more sense in comparison to ```stable``` than testing. However, my expectations could also be flawed. Do you remember why each of these testing lines were or'd with batched?